Gems/AWSCore/Code/Platform/Common/GetJsonSchemaPath_Common.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/IO/Path/Path.h>
+#include <AzCore/Utils/Utils.h>
+
+namespace AWSCore
+{
+    namespace Platform
+    {
+        AZ::IO::Path GetJsonSchemaPath()
+        {
+            static constexpr const char ResourceMapppingJsonSchemaFilePath[] =
+                "Gems/AWSCore/resource_mapping_schema.json";
+
+            AZ::IO::Path executablePath = AZ::IO::PathView(AZ::Utils::GetExecutableDirectory());
+            AZ::IO::Path jsonSchemaPath = (executablePath / ResourceMapppingJsonSchemaFilePath).LexicallyNormal();
+
```

Comment:
Why not just load this file from the Asset Cache in all cases?

Reply:
The AWSCore system component is activated even before the resource mapping file is processed by AP. This can be a problem when developers launch the Editor for the first time without all the assets pre-processed. For Android build, we have all the assets copied to the APK and this will not be an issue.